tests/test_acl.py

```diff
@@ -20,12 +20,13 @@
 __maintainer__ = 'Animesh Pathak <animesh@dgrpah.io>'
 
 import logging
-import os
+import shutil
 import unittest
 
 from . import helper
 import pydgraph
 
+@unittest.skipIf(shutil.which('dgraph') is None, 'Dgraph binary not found.')
```

Comment:
why are we adding this?

Reply:
this test requires the dgraph binary to be located on the client's path

Reply:
But this is the kinda thing that ends up in CI not running the tests at all. We should really consider undoing it.